### DISCUSSION TITLE
Let's discuss about open PR's 🧾 #39

### DISCUSSION THREAD

**Discussion by externalPointerVariable** [AryaChakraborty/articulus_rag#54] [General]
Hey @AryaChakraborty ,
I initiated a pull request about a week ago but no action is been taken over it ever since. I am referring to pull request #39 . I have not received any updates regarding that PR. Please look into it and start the review process. All the details regarding "What I have done ?" is discussed in the PR I have tagged and if you want more information feel free to discuss

**Comment by AryaChakraborty** [comment]
Hi @thakurboy004 the branch has merge conflicts, please look into them.

**Reply by externalPointerVariable** [reply]
Okay I'll look into it

**Reply by externalPointerVariable** [reply]
I have resolved the occuring conflicts

**Reply by AryaChakraborty** [reply]
cool, will ask my team to review it

**Reply by externalPointerVariable** [reply]
okay looking forward to it
